akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2Blueprint.scala

```diff
@@ -51,6 +59,7 @@ object Http2Blueprint {
    */
   def httpLayer(): BidiFlow[HttpResponse, Http2SubStream, Http2SubStream, HttpRequest, NotUsed] = {
     val incomingRequests = Flow[Http2SubStream].via(new HttpRequestHeaderHpackDecompression)
+      .log(getClass.getSimpleName)
```

Comment:
No matter what, getSimpleName is not safe for Scala classes, hence Logging.simpleName